systemtests/network_test.go

```diff
@@ -266,6 +266,51 @@ func (s *systemtestSuite) testNetworkAddDeleteNoGateway(c *C, encap string) {
 	}
 }
 
+func (s *systemtestSuite) TestNetworkEPGAddDelete(c *C) {
+
+	var err error
+	numContainers := 4
+	networkName := "net1"
+	epgName := "epg1"
+
+	for i := 0; i < s.iterations; i++ {
+		// Create Endpoints
+		network := &client.Network{
```

Comment:
Also should this test be run in L3 mode as well ? If so we need to add setup bgp step